frontend/src/scenes/retention/RetentionModal.tsx

```diff
@@ -72,7 +72,7 @@ export function RetentionModal({
                     {actors.missing_persons > 1
                         ? `${aggregationTargetLabel.plural} are`
                         : `${aggregationTargetLabel.singular} is`}{' '}
-                    not shown because they've been lost.{' '}
+                    not shown because they've been merged with those listed, or deleted.{' '}
                     <a href="https://posthog.com/docs/how-posthog-works/queries#insights-counting-unique-persons">
                         Read more here for when this can happen
```

Comment:
Maybe just "Read more" to make it shorter and less confusing too

Reply:
<img width="563" alt="Screenshot 2022-11-14 at 20 00 09" src="https://user-images.githubusercontent.com/4550621/201743566-02d79dcd-8908-4a98-8c0e-b8af9d49f928.png">